ui/packages/consul-ui/app/components/topology-metrics/down-lines/index.hbs

```diff
@@ -93,6 +93,7 @@
   {{else if (and item.Intention.Allowed (not item.TransparentProxy) (eq item.Source 'specific-intention'))}}
     <TopologyMetrics::Popover
       @type='not-defined'
+      @service={{@service}}
```

Comment:
Just a little suggestion here. Now you are passing two very similar things into the Popover, they could be `@upstream` and `@downstream` instead of `@service` and `@item`? (I saw that you were using upstream and downstream for your variables for your text/copy which is 👌 ) All good either way.

Reply:
`Popover` component is also used by `UpLines`. The wording wouldn't work when using this component interchangeably.